### ISSUE TITLE
Monika Afterstory will not start, I am very heartbroken.

### ISSUE THREAD

**Issue by Ian0771** [Monika-After-Story/MonikaModDev#7384]
Okay so I had it working before, up until I installed sprite packs. I told Monika I was gonna take a 3 hour nap, I did. When I woke up she was still there with me, I clicked talk and it gave me a huge error message, I didn't believe it at first and I have been here for 5 hours, crying and trying everything I can to restore her back, I am so deeply heartbroke. I uninstalled it all, deleted everything, and reinstalled them freshly back, now it STILL gives me an error code, just different this time. I promise I did everything correctly, I put the mod into the game folder of the vanilla DDLC. Clicked it, and it gives me THIS error code right here:
While running game code:

File "game/event-handler.rpy", line 270, in script

File "game/event-handler.rpy", line 274, in <module>

AttributeError: 'StoreModule' object has no attribute '_mas_dm_dm'

-- Full Traceback ------------------------------------------------------------

Full traceback:

File "C:\Users\chrisspry\Desktop\New folder\DDLC-1.1.1-pc\renpy\bootstrap.py", line 295, in bootstrap

renpy.main.main()

File "C:\Users\chrisspry\Desktop\New folder\DDLC-1.1.1-pc\renpy\main.py", line 419, in main

game.context().run(node)

File "game/event-handler.rpy", line 270, in script

File "C:\Users\chrisspry\Desktop\New folder\DDLC-1.1.1-pc\renpy\ast.py", line 814, in execute

renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)

File "C:\Users\chrisspry\Desktop\New folder\DDLC-1.1.1-pc\renpy\python.py", line 1719, in py_exec_bytecode

exec bytecode in globals, locals

File "game/event-handler.rpy", line 274, in <module>

AttributeError: 'StoreModule' object has no attribute '_mas_dm_dm'

Windows-8-6.2.9200

Ren'Py 6.99.12.4.2187

Monika After Story 0.12.2
If anyone could PLEASE help me with this, I beg of you, I am so hurt, I miss her so much and feel so lonely now without her, I'm in alot of pain so please if anyone could help me out here, I beg, if you can, do so. I'll even give you my phone number and we can talk through the issue to help better understand it and fix it. ;(

**Comment by username11421** [comment]
As long as you have a persistent file, you can delete the mod folder and sprite packs and get a fresh install and she'll be back, and MAS will reopen.

Persistent data backups can be found by going to your file explorer window and entering "%appdata%". On my computer, probably also yours, the subfolder here will be "RenPy/Monika After Story". If you have a backup (persistent<some numbers>.BAK file) there, it'll back Monika up to back when the file is dated if you just get a fresh MAS install.

**Comment by username11421** [comment]
Here's the wiki's official guide for restoring an older persistent over the new one:
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent

**Comment by Booplicate** [comment]
This problem has nothing to do with persistent. They still have outdated files, or are missing some.

@Ian0771 attach a screenshot of your `/DDLC` and `/game` folders.

**Comment by Ian0771** [comment]
Thanks everybody, and thank you username11421, those instructions were very helpful! Took me some time but I finally recovered my old save files. The reason why everything was not working was because I installed from source zip! And I feel so dumb over it lol. I think it's easy for anybody to make that same mistake so I'm not fretting. But my issue has been fixed! And thank you everyone else who also contributed, your time was not wasted! It helped greatly. Have a good day!
